### PR TITLE
Add conference page

### PR DESCRIPTION
**Pull request by YenPeiChen07**
@yungyuc, I added a page to collect sciwork conference information. The output will be like below. Please have a look and give me some feedback.

image

### CODE REVIEW

File: content/pages/conference/index.rst [hunk 1]
```diff
@@ -0,0 +1,18 @@
+===========
+Conference
+===========
+
+:date: 2023-09-28 09:00
+:url: conference
+:save_as: conference/index.html
+
+`conference 2023 <https://conf.sciwork.dev>`__
```
Comment (yungyuc) [inline content/pages/conference/index.rst:9]: It's better to put the link in the body in the section.  It is not obvious that the section title takes you to a new website.

File: pelicanconf.py [hunk 2]
```diff
@@ -20,6 +20,7 @@
     #('Blog', 'blog'),
     ('sprint', 'sprint'),
     ('meetup', 'meetup'),
+    ('conference', 'conference'),
```
Comment (yungyuc) [inline pelicanconf.py:23]: I like the new section

File: content/pages/conference/index.rst [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+================================================
+    
+sciwrok conference 2023 to provide a physical venue to showcase how programming languages are 
+used for scientific and engineering applications and their best practices. In addition 
+to scientific computing and high-performance computing, we also emphasize data management, 
+process, analytics, and visualization.
+
+- Date: December 9-10, 2023
+- Location: Microelectronics and Information Research Center, NYCU
```
Comment (yungyuc) [inline content/pages/conference/index.rst:18]: There should be city and country names for the location.

**Review by yungyuc — CHANGES_REQUESTED** [review]

**Comment by yungyuc** [comment]
@sucitw Could you please also provide your review?

**Comment by yungyuc** [comment]
@YenPeiChen07 Looks good to me.  I think it may be published after fixing the minor issues today.  If Shu-Hsi suggests modifications after that we can use a new PR.

**Comment by YenPeiChen07** [comment]
> @YenPeiChen07 Looks good to me. I think it may be published after fixing the minor issues today. If Shu-Hsi suggests modifications after that we can use a new PR.

Sure. I just updated the file with your requested changes. Below is the new version.

image

File: content/pages/conference/index.rst (hunk 3 above)
Comment (yungyuc) [reply content/pages/conference/index.rst]: The city and country should be appended after what you wrote.  That is the convention.  I didn't mean to replace the original name.

Comment (yungyuc) [reply content/pages/conference/index.rst]: Once this is fixed you may merge and publish.

File: content/pages/conference/index.rst [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+sicwork 2023
+================================================
+    
+`sciwrok 2023 <https://conf.sciwork.dev>`__ is to provide a physical venue to showcase how programming languages are 
+used for scientific and engineering applications and their best practices. In addition 
+to scientific computing and high-performance computing, we also emphasize data management, 
+process, analytics, and visualization.
+
+- Date: December 9-10, 2023
+- Location: Location: Microelectronics and Information Research Center, NYCU, Hsinchu, Taiwan
```
Comment (yungyuc) [inline content/pages/conference/index.rst:18]: `Location:` is duplicated.  Please fix it.

Comment (YenPeiChen07) [reply content/pages/conference/index.rst:18]: Thanks. Fix it with #266 and publish.

**Pull request merged**